public/js/app.js

```diff
@@ -1,106 +1,82 @@
-$(function() {
-    Timer.init();
+var app = angular.module('app', []);
```

Comment:
Angular files are generally structured in a specific way, check that link I showed you above.  Also generally you would put your app, ctrls and services in different files.  Its a little overkill for a project of this size, but then again u want to show you know these things.